.Rproj.user/3A98825C/pcs/source-pane.pper

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Oops, it seems that .Rproj.user files are not properly ignored. When you see them staged, you may do:

```
git rm --cached -r .
git add .
git update-index --assume-unchanged
git commit -a -m "forget cached"
```

https://github.com/forestgeo/learn/issues/5